ocs_ci/templates/CSI/rbd/storageclass.yaml

```diff
@@ -15,8 +15,10 @@ parameters:
     # RBD image format. Defaults to "2".
     imageFormat: "2"
 
-    # RBD image features. Available for imageFormat: "2". CSI RBD currently supports only `layering` feature.
-    imageFeatures: layering
+    # RBD image features. Available for imageFormat: "2".
+    # CSI RBD currently supports `layering`, `journaling`, `exclusive-lock`,
+    # `object-map`, `fast-diff`, `deep-flatten` features.
+    imageFeatures: "layering,deep-flatten,exclusive-lock,object-map,fast-diff"
```

Comment:
We need to handle this via code we cannot use this directly because this yaml is used on older release as well

Reply:
Missed this case, Thanks for catching it. Done, PTAL